jenkinsfiles/ginkgo-kernel.Jenkinsfile

```diff
@@ -193,6 +193,7 @@ pipeline {
 
     post {
         always {
+            sh 'lscpu'
```

Comment:
Don't we need to the same for `ginkgo-kubernetes-all.Jenkinsfile` and `ginkgo.Jenkinsfile`?